validation-test/ParseableInterface/verify_all_overlays.py

```diff
@@ -6,9 +6,10 @@
 # RUN: %empty-directory(%t)
 # RUN: %{python} %s %target-os %target-cpu %platform-sdk-overlay-dir %t \
 # RUN:   %target-swift-frontend -build-module-from-parseable-interface \
-# RUN:     -Fsystem %sdk/System/Library/PrivateFrameworks/ >> %t/failures.txt
+# RUN:     -Fsystem %sdk/System/Library/PrivateFrameworks/ | sort > %t/failures.txt
```

Comment:
I think I had `>>` originally because that kept it from creating the file when parseable interfaces were disabled, or if there are no failures at all. Looking at the test again, though, I'm not sure why I did that. Diffing two empty files is fine.

Reply:
I've made similarly confusing choices myself too ... sometimes it requires a bit more thinking to use the "simple" solution because the complex solution is more obvious :-)